### ISSUE TITLE
confirm `fmap` without suffix/label sufficent for heuiconv reproin

### ISSUE THREAD

**Issue by WillForan** [NPACore/ReproIn-Transition-Example#2]

**Comment by WillForan** [comment]
We need a "no suffix (multiple series number output from single sequence)" option for the `fmap` DataType such that the suffix portion of the name is entirely skipped.

With fmap + no suffix selected, the constructed sequence name should be like  `fmap_acq-dwi` 

![image](https://github.com/user-attachments/assets/66c2d385-494a-46b0-89a5-3f659063425f)


The need for this is described in https://github.com/nipy/heudiconv/issues/763. One scanner sequence creates generates dicoms with 2 series numbers (which becomes 2 folders from one acquisition).

![](https://private-user-images.githubusercontent.com/680742/351103727-874d9892-3469-4839-a825-17ccfd0d5415.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NDg2MDEsIm5iZiI6MTcyMjQ0ODMwMSwicGF0aCI6Ii82ODA3NDIvMzUxMTAzNzI3LTg3NGQ5ODkyLTM0NjktNDgzOS1hODI1LTE3Y2NmZDBkNTQxNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDczMVQxNzUxNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OGU4ZTUzYWEyZjgwMTA4ZDJiYzU3NGNlMGQ3NGM4OGM3NDZjOTlmZWI1NDFlZmU2YTQ2YmY4Nzg4ZTc2MzVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.agAeWyPILeVk5Q2ZOmZ0UC3pPtkRWRDD2UMq7IGdoFI)


The change need is likely to be within the `fill_suffix_opts` function with a specific case if `dt == "fmap"` prepend `entity_table[dt]` with a copy of e.g. magnitude but with `description`, `display_name`, and `value` change.
https://github.com/NPACore/reproin-namer/blob/main/entity_ux/index.html#L87

[`update_name`](https://github.com/NPACore/reproin-namer/blob/main/entity_ux/index.html#L186) might also need to be modified to leave `reproin_name` untouched when `value` is the newly implemented no suffix option.

![image](https://github.com/user-attachments/assets/0510daca-069f-4c5b-93bc-f5e187d28a92)

**Comment by WillForan** [comment]
`make` in this repo will download phantom dicoms to `examples/dicom/{5,6}`, modify the header w/copies in `dcm-rehead`, and run heudiconv on them to create `bids/`

```
git clone https://github.com/NPACore/reproin-namer
cd reproin-namer
make # will create virtualenv; but heudiconv needs dcm2niix
```

AFNI's `dicom_hdr` (verbose) and `dicom_hinfo` (terse) are good places to inspect the example dicoms. The python code uses `pydicom` to get at these values.

```
dicom_hdr examples/dicom/5/TESTBR.MR.DEV-SCHIRDA_041119_BRAINO.0005.0001.2020.07.18.21.25.34.768215.1213287301.IMA examples/dicom/6/TESTBR.MR.DEV-SCHIRDA_041119_BRAINO.0006.0001.2020.07.18.21.25.34.768215.1213288871.IMA|grep -P 'Series.Num|Protocol.Name'

  0018 1030       36 [1674    ] //              ACQ Protocol Name//gre_2D_28sls_3mm_3constrasts_fov210 
  0020 0011        2 [2182    ] //              REL Series Number//5 
  
  0018 1030       36 [1674    ] //              ACQ Protocol Name//gre_2D_28sls_3mm_3constrasts_fov210 
  0020 0011        2 [2182    ] //              REL Series Number//6 


dicom_hinfo -tag 0020,0011 -no_name examples/dicom/*/TESTBR.MR.DEV-SCHIRDA_041119_BRAINO.000[56].0001.*.IMA
  5
  6

 python -c 'from pydicom import read_file; print(read_file("examples/dicom/5/TESTBR.MR.DEV-SCHIRDA_041119_BRAINO.0005.0001.2020.07.18.21.25.34.768215.1213287301.IMA").SeriesNumber)'
  5
```

this repo is also a python package that provides `dicom-rewrite-pname` to rename the header (demonstrated in the [`Makefile`](https://github.com/NPACore/reproin-namer/blob/main/Makefile#L22))

```

# header name changed as run by 'make'
#   dicom-rewrite-pname -o dcm-rehead/5 -n fmap_acq-dwi examples/dicom/5/*

dicom_hdr dcm-rehead/5/TESTBR.MR.DEV-SCHIRDA_041119_BRAINO.0005.0001.2020.07.18.21.25.34.768215.1213287301.IMA|grep -P 'Series.Num|Protocol.Name'

0018 1030       12 [1674    ] //              ACQ Protocol Name//fmap_acq-dwi
0020 0011        2 [2158    ] //              REL Series Number//5
```

**Comment by WillForan** [comment]
part we control (UI builder) is resolved by c9462f252783fc480336dfe6be0ae32b9c65bbe9
